src/components/ebay-chips-combobox/marko-tag.json

```diff
@@ -5,43 +5,17 @@
         "type": "expression"
     },
     "@html-attributes": "expression",
-    "@borderless": "boolean",
-    "@autocomplete": "string",
     "@expanded": "boolean",
-    "@list-selection": {
-        "enum": ["automatic", "manual"]
-    },
-    "@disabled": "#html-disabled",
-    "@form": "#html-form",
-    "@name": "#html-name",
-    "@placeholder": "#html-placeholder",
-    "@required": "#html-required",
-    "@value": "#html-value",
-    "@role": "never",
-    "@aria-autocomplete": "never",
-    "@aria-haspopup": "never",
-    "@floating-label": "string",
-    "@actionable <actionable>": {
+    "@fluid": "boolean",
+    "@error": "boolean",
+    "@list-selection": "string",
+    "@option <option>[]": {
```

Comment:
I think this needs to be 
`"@options <option>[]": {`
Then in the code we should reference `input.options`
See https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-combobox/marko-tag.json#L32
`@options` is the name of the input when you get it in code, where as the `<option>` part is the name of the attribute tag.